dev-libs/crypto++/crypto++-8.2.0.recipe

```diff
@@ -86,8 +84,7 @@ Version: $portVersion
 Libs: -lcryptopp
 EOF
 
-	packageEntries devel \
-		$relativeDevelopDir
+	packageEntries devel $relativeDevelopDir
```

Comment:
Is $reletiveDevelopDir needed here? Or could you just use $developDir?

Reply:
Well, I don't know what is the differences between $relativeDevelopDir
and $developDir.
I tried build without this line and I got error, so just leave there.
Can you help me, where can I found this descriptions? Should I check the haikuporter source?

Reply:
Ok, I found in the haikuporter's wiki. Absolut path vs relative path.
So:
developDir=/boot/system/develop
relativeDevelopDir=/develop ?

Reply:
From the wiki "$relativeDevelopDir: $developDir as a relative directory (to prefix)." https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes, haven't checked if it works with $developDir :)

Reply:
Most invocations of `packageEntries` use $developDir, so try that.

Reply:
Done.